### PR TITLE
[enhancement](delete) print all debug mode session variables on error

### PR DESCRIPTION
**Pull request by zhannngchen**
## Proposed changes

Issue Number: close #xxx

some user may got error "Delete is forbidden since current session is in debug mode" while run delete, we'd better print the debug variables to make user locate the issue easier.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/UpdateCommand.java [hunk 1]
```diff
@@ -223,7 +223,7 @@ private void checkTable(ConnectContext ctx) {
         if (ctx.getSessionVariable().isInDebugMode()) {
             throw new AnalysisException("Update is forbidden since current session is in debug mode."
                     + " Please check the following session variables: "
-                    + String.join(", ", SessionVariable.DEBUG_VARIABLES));
+                    + ConnectContext.get().getSessionVariable().printDebugModeVariables());
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/UpdateCommand.java:226]:
```suggestion
                    + ctx.getSessionVariable().printDebugModeVariables());
```

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/DeleteFromUsingCommand.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
         if (ctx.getSessionVariable().isInDebugMode()) {
             throw new AnalysisException("Delete is forbidden since current session is in debug mode."
                     + " Please check the following session variables: "
-                    + String.join(", ", SessionVariable.DEBUG_VARIABLES));
+                    + ConnectContext.get().getSessionVariable().printDebugModeVariables());
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/DeleteFromUsingCommand.java:78]:
```suggestion
                    + ctx.getSessionVariable().printDebugModeVariables());
```

**Comment by zhannngchen** [comment]
run buildall

**Comment by zhannngchen** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 3]
```diff
@@ -2046,6 +2046,33 @@ public boolean isInDebugMode() {
         return showHiddenColumns || skipDeleteBitmap || skipDeletePredicate || skipDeleteSign || skipStorageEngineMerge;
     }
 
+    public String printDebugModeVariables() {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:2049]: Naming `getDebugModeVariables()`

**Review by gavinchou — COMMENTED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java (hunk 3 above)
Comment (zhannngchen) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:2049]: same style like existing functions, like `printFuzzyVariables()`

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**